src/js/views/patients/patient/flow/patient-flow.scss

```diff
@@ -26,7 +26,7 @@
   background: $white;
   color: $black-20;
   line-height: 1;
-  padding: 16px 0 8px 0;
+  padding: 16px 0 16px;
```

Comment:
This update is to align the flow title and patient name (in the patient sidebar) on the flow page.

![align-patient-name-with-flow-title](https://user-images.githubusercontent.com/35355575/173112639-8d956264-a0ae-4b48-9138-6675efbfa4c4.png)